templates/rails/spec/javascript/support/enzyme.js

```diff
@@ -1,4 +1,4 @@
 const Enzyme = require('enzyme');
-const EnzymeAdapter = require('enzyme-adapter-react-15.4');
+const EnzymeAdapter = require('enzyme-adapter-react-16');
```

Comment:
we may want to make this a constant that is easy to update. that way, we can orchestrate the adapter upgrade in `package.json` and here with a one line change. wdyt?

I can foresee us bumping the adapter but forgetting to update this file.

Reply:
wise! 🙇‍♀I'll make that change and send this your way again